main.py

```
@@ -237,6 +323,319 @@ def set_hue_diag(self, enabled: bool):
+                applied.append("idle_state")
+            mz = theme.get("mz_startlights")
+            if isinstance(mz, dict) and "direction" in mz and "mode" in mz:
+                self.runner.set_mz_startlights(mz["direction"], mz["mode"])
+                applied.append("mz_startlights")
+            if isinstance(theme.get("rpm_gradient"), list):
+                self.runner.set_rpm_gradient(theme["rpm_gradient"])
+                applied.append("rpm_gradient")
+            if isinstance(theme.get("curves"), dict):
+                self.runner.set_curves(theme["curves"])
+                applied.append("curves")
+        except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)